uamqp/client.py

```diff
@@ -218,6 +234,11 @@ def auth_complete(self):
 
         :rtype: bool
         """
+        if self._cbs_authenticator:
+            if self._cbs_authenticator.state.value != 2 or not self._cbs_authenticator.handle_token():
```

Comment:
move the `_cbs_authenticator` check into the `handle_token` part? or at least improve the cbs_authenticator state to use Enums:
```
class CbsAuthenticatorState(Enum):
    Ok = 1
    Error = ...
```

Reply:
Yeah I think the state check could be moved into `handle_token`

Reply:
code updated ^^